ivy_tests/test_ivy/test_functional/test_core/test_creation.py

```diff
@@ -58,13 +59,14 @@ def test_native_array(
 @given(
     dtype_and_start_stop=helpers.dtype_and_values(
         available_dtypes=ivy_np.valid_numeric_dtypes,
-        num_arrays=1,
-        allow_inf=False,
+        num_arrays=2,
+        min_value=None,
+        max_value=None,
         min_num_dims=1,
```

Comment:
we should diversify the shapes more, with more diversity in the dimensions and dimension sizes.

Reply:
yes of course